### PR TITLE
no Lass edits

### PR DESCRIPTION
**Pull request by Squarerootofsheep**
Fixed BC1 2nd Caterpie. Save 2 HAs for Moon Rocket added since Leer + Tacklex2(+PS) is terrible on Moocket Rat. Added teach thrash over wg (ty Grogir). Changed 0 potions left to 7 specs and 3 speeds along with adding when/how to use the speeds (purposefully omitted surf menu swap for simplicity). Added "after the fight" to para heal use when thrashing Silph Arbok. Changed VR Ryhorn to HD as per wsop's suggestion.

### CODE REVIEW

File: docs/gen-1/red-blue/main-glitchless/beginner-route/2021/lass/README.md [hunk 1]
```diff
@@ -458,7 +458,7 @@ NOTE: From this point on you have 2 revives which means deaths aren't as scary,
 - EQ x3
 - Elixer on Weezing
   - if Koga used x attack use Bubblebeam till you die
-  - if you didn’t die from koga you can still try to get red bar by following the HP strats for the [backups](https://pastebin.com/HgyuMvyr). Note that this can be risky
+  - if you didn’t die from koga you can still try to get red bar by following the HP strats for the [backups](https://pastebin.com/2DBBnkmf). Note that this can be risky
```
Comment (stocchi) [inline docs/gen-1/red-blue/main-glitchless/beginner-route/2021/lass/README.md]: why this change? this pastebin is old and doesnt even have boom strats

**Review by stocchi — CHANGES_REQUESTED** [review]

File: docs/gen-1/red-blue/main-glitchless/beginner-route/2021/lass/README.md (hunk 1 above)
Comment (hwangbro) [reply docs/gen-1/red-blue/main-glitchless/beginner-route/2021/lass/README.md]: This PR has old commits in it and doesn't represent the real changes being made yet.

**Review by stocchi — APPROVED** [review]
